### PR TITLE
Update argparse.pyi

### PR DESCRIPTION
**Pull request by rmcgibbo**
argparse.Namespace is also useful as a Bunch

### CODE REVIEW

File: stdlib/2and3/argparse.pyi [hunk 1]
```diff
@@ -117,6 +117,7 @@ class Action:
                  option_string: _Text = ...) -> None: ...
 
 class Namespace:
+    def __init__(self, **kwargs: Dict[_Text, Any]) -> None: ...
```
Comment (JelleZijlstra) [inline stdlib/2and3/argparse.pyi]: This means that every keyword argument must be a Dict. I think you want just `**kwargs: Any`. (That would also fix the Travis failures.)

**Comment by rmcgibbo** [comment]
Oh, that makes sense. Thanks for the tip!

File: stdlib/2and3/argparse.pyi [hunk 2]
```diff
@@ -117,6 +117,7 @@ class Action:
                  option_string: _Text = ...) -> None: ...
 
 class Namespace:
+    def __init__(self, **kwargs: Any) -> None: ...
     def __getattr__(self, name: _Text) -> Any: ...
     def __setattr__(self, name: _Text, value: Any) -> None: ...
```
Comment (JelleZijlstra) [inline stdlib/2and3/argparse.pyi]: Looking at the code I noticed it also has `def __contains__(self, key: str) -> bool: ...`; would you mind adding that?

Comment (rmcgibbo) [reply stdlib/2and3/argparse.pyi]: done

Comment (JelleZijlstra) [reply stdlib/2and3/argparse.pyi]: Thanks! I'll merge once Travis passes (unless I forget).

**Pull request merged**